data/2015/colleges/Business Administration Main Page.xhtml

```diff
@@ -614,68 +613,77 @@
-<p class="list-indent">Community and Regional Planning</p>
-<p class="list-indent">Computer Science</p>
-<p class="list-indent">Conflict and Conflict Resolution</p>
-<p class="list-indent">Criminal Justice</p>
-<p class="list-indent">Czech <span class="list-ital">(see Modern Languages)</span></p>
-<p class="list-indent">Dance</p>
-<p class="list-indent">Education</p>
-<p class="list-indent">English</p>
-<p class="list-indent">Environmental Studies</p>
-<p class="list-indent">Ethnic Studies</p>
-<p class="list-indent">Film Studies</p>
-<p class="list-indent">French <span class="list-ital">(see Modern Languages)</span></p>
-<p class="list-indent">Geography</p>
-<p class="list-indent">Geology</p>
-<p class="list-indent">German <span class="list-ital">(see Modern Languages)</span></p>
-<p class="list-indent">Gerontology</p>
-<p class="list-indent">Global Studies</p>
-<p class="list-indent">Great Plains Studies</p>
-<p class="list-indent">Greek <span class="list-ital">(see Classics &amp; Religious Studies)</span></p>
-<p class="list-indent">History</p>
-<p class="list-indent">Human Rights and Human Diversity</p>
-<p class="list-indent">Individualized Program of Studies</p>
-<p class="list-indent">Japanese</p>
-<p class="list-indent">Jewish Studies</p>
-<p class="list-indent">Latin <span class="list-ital">(see Classics &amp; Religious Studies)</span></p>
-<p class="list-indent">Latina/Latino Studies <span class="list-ital">(see U.S. Latina/Latino Studies)</span></p>
-<p class="list-indent">Mathematics</p>
-<p class="list-indent">Medieval and Renaissance Studies</p>
-<p class="list-indent">Meteorology-Climatology <span class="list-ital">(see Geosciences)</span></p>
-<p class="list-indent">Military Science</p>
-<p class="list-indent">Music</p>
-<p class="list-indent">Native American Studies</p>
-<p class="list-indent">Philosophy</p>
-<p class="list-indent">Physics</p>
-<p class="list-indent">Political Science</p>
-<p class="list-indent">Psychology</p>
-<p class="list-indent">Public Policy Analysis and Evaluation</p>
-<p class="list-indent">Religious Studies <span class="list-ital">(see Classics &amp; Religious Studies)</span></p>
-<p class="list-indent">Russian <span class="list-ital">(see Modern Languages)</span></p>
-<p class="list-indent">Sociology</p>
-<p class="list-indent">Spanish <span class="list-ital">(see Modern Languages)</span></p>
-<p class="list-indent">Statistics</p>
-<p class="list-indent">Textiles, Merchandising and Fashion Design</p>
-<p class="list-indent">Theatre Arts</p>
-<p class="list-indent">U.S. Latina/Latino Studies</p>
-<p class="list-indent">University Studies</p>
-<p class="list-indent">Women’s and Gender Studies</p>
+<p class="MsoNormal" style="margin-bottom: .0001pt; line-height: normal;"><span style="mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: 'Times New Roman'; color: black;">Aerospace Studies</span></p>
```

Comment:
All of the remain paragraphs need to get reverted to the list-indent style. copy/paste FAIL!